src/include/migraphx/module.hpp

```diff
@@ -23,6 +24,8 @@ struct module_impl;
 
 using parameter_map = std::unordered_map<std::string, argument>;
 
+using module_ref = module*;
```

Comment:
This is already declared in the module_ref header.

Reply:
removed.